Disable the languages, which don't have yet a path in /usr/share/locale/

Index: tar-1.23/po/LINGUAS
===================================================================
--- tar-1.23.orig/po/LINGUAS
+++ tar-1.23/po/LINGUAS
@@ -16,7 +16,6 @@ id
 it
 ja
 ko
-ky
 ms
 nb
 nl
